app/src/main/res/values/strings.xml

```diff
@@ -378,7 +378,7 @@ TODO: error and download_document_confirm_prefix need to be parameterized rather
     <string name="backup_success">My Notes and Bookmarks saved in %s on the SD card</string>    
     <string name="restore">Restore</string>
```

Comment:
Maybe menu item should be "Restore...", and likewise "Backup...", to indicate that another screen will come up?

Also, though not directly related to this PR, I think the user should be asked to confirm that he/she wants to restore and replace his current database (which will include notes, bookmarks, and reading plans)

Reply:
Fixed both